### ISSUE TITLE
`Missings` are missing from the list of dependencies

### ISSUE THREAD

**Issue by dmitrijsc** [JuliaDatabases/SQLite.jl#148]
Hello

I've been receiving a message that `Missings` are missing from the list of dependencies and could not compile SQLite.

I had to edit the Manifest file in order to get it running: v"0.7.0-beta2.0" (brand new setup)

Sorry, don't have an exact error message.

**Comment by pstaabp** [comment]
I want to second this.  Here's the error I'm getting on the just-released v0.7:
```
Warning: Package SQLite does not have Missings in its dependencies:
│ - If you have SQLite checked out for development and have
│   added Missings as a dependency but haven't updated your primary
│   environment's manifest file, try `Pkg.resolve()`.
│ - Otherwise you may need to report an issue with SQLite
└ Loading Missings into SQLite from project dependency, future warnings for SQLite are suppressed.
```

Also, there are a number of deprecation warnings in the `src/api.jl` and `src/SQLite.jl` files.

**Comment by quinnj** [comment]
Fixed on master
